spec/order_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'order'
+
+describe Order do
+
+  subject(:order)       { Order.new(menu) }
+  let(:menu)            { double(:menu, :dishes => { "Tom Yum" => 9.90, "Pad Thai" => 10.00, "Green Curry" => 6.99, "Red Curry" => 6.00, "Aloe Drink" => 2.00, "Chocolate Ice Cream" => 4.00, "California Roll" => 12.00 }) }
```

Comment:
This looks as though you've got the hang of doubles 👍 

Your tests all look to be using them correctly too.